diff --git a/node_modules/react-native/scripts/github-actions.js b/node_modules/react-native/scripts/github-actions.js
new file mode 100644
index 0000000..d48bb8b
--- /dev/null
+++ b/node_modules/react-native/scripts/github-actions.js
@@ -0,0 +1,11 @@
+// @ts-check
+
+const execa = require('execa');
+const fs = require('fs');
+
+const DEST = `${process.env.CONFIGURATION_BUILD_DIR}/${process.env.UNLOCALIZED_RESOURCES_FOLDER_PATH}`
+execa.sync(`npx react-native bundle --entry-file='index.js' --bundle-output='${DEST}/main.jsbundle' --dev=false --platform='ios' --assets-dest='${DEST}'`, { shell: true, stdio: 'inherit' });
+
+if (!fs.existsSync(`${DEST}/main.jsbundle`)) {
+  throw new Error(`main.jsbundle Missing !`)
+}
